### PR TITLE
Truncate postal code with 5 digits

### PR DESCRIPTION
**Pull request by pmargreff**
Some listings would be not exported to portals due to the invalid postal code. Google maps probably return some postal codes in a 5 digits format (old and invalid format), on these cases we are now truncating "-000" (nevertheless it does not reflect the generated code be the correct one). 

While digging to this I found [we removed the validation on zip code some time ago](https://github.com/emcasa/backend/commit/114c14a4cf094adaa9255137d782a40f7c083b6f) (probably duo the same issue), should it be reinserted now all the know cases are formatted?

### CODE REVIEW

**Comment by rhnonose** [comment]
We should be careful with Google Map APIs return, a strict validation thing will error on the user and reduce our opportunity to fix manually and document exceptions.

For now it's fine to keep things without strict validation since we can more easily fix manually with the dashboard.

**Comment by pmargreff** [comment]
> For now it's fine to keep things without strict validation since we can more easily fix manually with the dashboard.

Ok, it looks reasonable to keep things like this. :+1:

File: apps/re/lib/addresses/address.ex [hunk 1]
```diff
@@ -55,4 +56,22 @@ defmodule Re.Address do
   def generate_slugs(changeset) do
     Enum.reduce(@sluggified_attr, changeset, &Slugs.generate_slug(&1, &2))
   end
+
+
+  def truncate_postal_code(changeset) do
```
Comment (joaodubas) [inline apps/re/lib/addresses/address.ex]: I think this operation isn't truncate, you're expanding the postal code to have 8 chars.

Comment (pmargreff) [reply apps/re/lib/addresses/address.ex]: Renamed the function based in [`String.pad_trailing/3`](https://hexdocs.pm/elixir/String.html#pad_trailing/3). 

Is this more meaningful? If it isn't I'm open for suggestions.

Comment (joaodubas) [reply apps/re/lib/addresses/address.ex]: Good one @pmargreff LGTM 👍

File: apps/re/test/addresses/address_test.exs [hunk 2]
```diff
@@ -74,9 +74,11 @@ defmodule Re.AddressTest do
     assert changeset.errors == [postal_code: {"has already been taken", []}]
   end
 
-  test "five digits posta code should be valid" do
+  test "five digits postal code should be truncated to eight digits" do
```
Comment (joaodubas) [inline apps/re/test/addresses/address_test.exs]:
```suggestion
  test "five digits postal code should be padded to eight digits" do
```

Comment (pmargreff) [reply apps/re/test/addresses/address_test.exs]: Thank you, I missed the change here.

**Pull request merged**